--- a/src/calendar_notes.h
+++ b/src/calendar_notes.h
@@ -52,7 +52,7 @@
 };
 
 gboolean    cal_check_note              (guint32 julian, GUI *appGUI);
 gboolean    cal_check_notes             (guint32 julian_start, guint32 julian_end, GUI *appGUI);
-void        cal_notes_foreach           (guint32 julian_start, guint32 julian_end, gboolean (*cnfunc)(), GUI *appGUI);
+void        cal_notes_foreach           (guint32 julian_start, guint32 julian_end, gboolean (*cnfunc)(struct note*, GUI*), GUI *appGUI);
 
 gchar *     cal_get_note                (guint32 julian, GUI *appGUI);
 gchar *     cal_get_note_color          (guint32 julian, GUI *appGUI);
--- a/src/calendar_notes.c
+++ b/src/calendar_notes.c
@@ -41,7 +41,7 @@
 }
 
 void
-cal_notes_foreach (guint32 julian_start, guint32 julian_end, gboolean (*cnfunc)(), GUI *appGUI)
+cal_notes_foreach (guint32 julian_start, guint32 julian_end, gboolean (*cnfunc)(struct note*, GUI*), GUI *appGUI)
 {
 	GSList *node;
 	struct note *n;
@@ -49,6 +49,6 @@
 		n = (struct note *) node->data;
 
 		if (n->date >= julian_start && n->date <= julian_end)
-			if ((*cnfunc)(n, appGUI) == TRUE)
+			if (cnfunc(n, appGUI) == TRUE)
 				break;
 	}
 }
