From 578105da4f7cd0a64f6de02e16177071d861f571 Mon Sep 17 00:00:00 2001
From: Naz <ndpm13@ch-naseem.com>
Date: Fri, 1 May 2026 10:45:05 +0100
Subject: [PATCH] fix: replace variable with hardcoded system paths

---
 res/config.ini | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/res/config.ini b/res/config.ini
index 86f44b4..e883fdf 100644
--- a/res/config.ini
+++ b/res/config.ini
@@ -102,13 +102,13 @@ border_fg = 0x00FFFFFF
 box_title = null
 
 # Brightness decrease command
-brightness_down_cmd = $PREFIX_DIRECTORY/bin/brightnessctl -q -n s 10%-
+brightness_down_cmd = /usr/bin/brightnessctl -q -n s 10%-
 
 # Brightness decrease key combination, or null to disable
 brightness_down_key = F5
 
 # Brightness increase command
-brightness_up_cmd = $PREFIX_DIRECTORY/bin/brightnessctl -q -n s +10%
+brightness_up_cmd = /usr/bin/brightnessctl -q -n s +10%
 
 # Brightness increase key combination, or null to disable
 brightness_up_key = F6
@@ -151,7 +151,7 @@ custom_bind_width = null
 # Custom sessions directory
 # You can specify multiple directories,
 # e.g. $CONFIG_DIRECTORY/ly/custom-sessions:$PREFIX_DIRECTORY/share/custom-sessions
-custom_sessions = $CONFIG_DIRECTORY/ly/custom-sessions
+custom_sessions = /etc/ly/custom-sessions
 
 # Input box active by default on startup
 # Available inputs: info_line, session, login, password
@@ -173,7 +173,7 @@ doom_middle_color = 0x00C78F17
 doom_bottom_color = 0x00FFFFFF
 
 # Dur file path
-dur_file_path = $CONFIG_DIRECTORY/ly/example.dur
+dur_file_path = /etc/ly/example.dur
 
 # Dur file alignment
 # The dur file can be aligned with a direction and centered easily with the flags below
@@ -328,7 +328,7 @@ service_name = ly
 session_log = .local/state/ly-session.log
 
 # Setup command
-setup_cmd = $CONFIG_DIRECTORY/ly/setup.sh
+setup_cmd = /etc/ly/setup.sh
 
 # Show the shell session in the session list
 # If false, the shell session will be hidden
@@ -356,7 +356,7 @@ sleep_key = F3
 
 # Command executed when starting Ly (before the TTY is taken control of)
 # See file at path below for an example of changing the default TTY colors
-start_cmd = $CONFIG_DIRECTORY/ly/startup.sh
+start_cmd = /etc/ly/startup.sh
 
 # Center the session name.
 text_in_center = false
@@ -373,11 +373,11 @@ vi_mode = false
 # You can specify multiple directories,
 # e.g. $PREFIX_DIRECTORY/share/wayland-sessions:$PREFIX_DIRECTORY/local/share/wayland-sessions
 # If null, Wayland sessions will not be shown
-waylandsessions = $PREFIX_DIRECTORY/share/wayland-sessions
+waylandsessions = /usr/share/wayland-sessions
 
 # Xorg server command
 # Add the -quiet argument to hide startup logs from the server
-x_cmd = $PREFIX_DIRECTORY/bin/X
+x_cmd = /usr/bin/X
 
 # Xorg virtual terminal number
 # Mostly useful for FreeBSD where choosing the current TTY causes issues
@@ -385,7 +385,7 @@ x_cmd = $PREFIX_DIRECTORY/bin/X
 x_vt = null
 
 # Xorg xauthority edition tool
-xauth_cmd = $PREFIX_DIRECTORY/bin/xauth
+xauth_cmd = /usr/bin/xauth
 
 # xinitrc
 # If null, the xinitrc session will be hidden
@@ -395,7 +395,7 @@ xinitrc = ~/.xinitrc
 # You can specify multiple directories,
 # e.g. $PREFIX_DIRECTORY/share/xsessions:$PREFIX_DIRECTORY/local/share/xsessions
 # If null, X11 sessions will not be shown
-xsessions = $PREFIX_DIRECTORY/share/xsessions
+xsessions = /usr/share/xsessions
 
 # Custom Commands and Labels:
 # The following examples below give an outline for setting up custom commands and labels.
-- 
2.54.0

